### ISSUE TITLE
Incorrect link on Project Migration page

### ISSUE THREAD

**Issue by ChrisMaddock** [MicrosoftDocs/visualstudio-docs#380]
On:
 
https://docs.microsoft.com/en-us/visualstudio/porting/port-migrate-and-upgrade-visual-studio-projects

There is the text: 

> If you don't see a project or file type listed here that should be, please consult the Visual Studio 2015 version of this topic and make a note in the comments below.

The link to the 'Visual Studio 2015 version of this topic' just links back to the same page.

**Comment by gewarren** [comment]
@ChrisMaddock thanks for reporting this. @Mikejo5000 can you take a look at fixing this?

**Comment by Mikejo5000** [comment]
@kraigb Do you own the porting docs?

**Comment by kraigb** [comment]
Yes, I do. That's supposed to be a link to MSDN; I'll check the history as to when it got changed.

**Comment by kraigb** [comment]
For some strange reason, the link in the master branch is incorrect; it was correct in the live branch so it's curious why and how it got to the live site. I also can't find when it got changed. In any case, I've restored it to where it should be with https://github.com/MicrosoftDocs/visualstudio-docs-pr/pull/1149. That change should go live tomorrow according to our publishing schedule.

In the meantime, the correct link to the VS2015 page is https://msdn.microsoft.com/library/hh266747.aspx.
